Automattic/wp-calypso
# Add Popup Jetpack Checkout Scheduling

#### Changes proposed in this Pull Request

##### Brief

Calendly offers a pretty good pop-up widget, that works around some aesthetic issues with the embedded widget.

* Modify Route for existing scheduling page to send to thank-you page and trigger widget
* Tweak thank-you page to make link a button.
* Trigger scheduling widget on button press.

![calypso localhost_3000_checkout_jetpack_thank-you_no-site_jetpack_backup_daily](https://user-images.githubusercontent.com/2810519/126724206-48664503-8e14-42c6-9f54-118a65b599e4.png)


#### Testing instructions



* Navigate to `checkout/jetpack/thank-you/no-site/jetpack_backup_daily` on branch
* Verify the "Schedule a 15 minute call now" link is now a button that triggers a pop-up widget one the same page, as in the screenshot above.
* Navigate to `/checkout/jetpack/schedule-happiness-appointment` on branch
* Verify you are taken to the same thank you page, but that the scheduling widget immediately opens
